server/text_generation_server/layers/marlin.py

```diff
@@ -215,6 +214,116 @@ def forward(self, A: torch.Tensor) -> torch.Tensor:
         return C
 
 
+GPTQ_MARLIN_24_MIN_THREAD_N = 128
+GPTQ_MARLIN_24_MIN_THREAD_K = 128
+GPTQ_MARLIN_24_MAX_PARALLEL = 64
+GPTQ_MARLIN_24_SUPPORTED_NUM_BITS = [4, 8]
+GPTQ_MARLIN_24_SUPPORTED_GROUP_SIZES = [-1, 128]
```

Comment:
This is coming from marlin directly I guess ?

Reply:
Yes, they come from upstream.